example/podpvc.yaml

```
@@ -64,3 +64,31 @@ spec:
     - name: my-volume
       persistentVolumeClaim:
         claimName: topolvm-pvc-thin
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: my-pod-ephemeral
+  namespace: default
+  labels:
+    app.kubernetes.io/name: my-pod-ephemeral
+    app: example
+spec:
+  containers:
+  - name: ubuntu
+    image: quay.io/cybozu/ubuntu:20.04
+    command: ["/usr/local/bin/pause"]
+    volumeMounts:
+    - mountPath: /test1
+      name: my-volume
+  volumes:
+    - name: my-volume
```

Please lower the indent by two. (Subsequent lines are similar.)
```suggestion
  - name: my-volume
```